pkg/controller/csr_check.go

```diff
@@ -361,23 +360,21 @@ func isApproved(csr *certificatesv1beta1.CertificateSigningRequest) bool {
 	return false
 }
 
-func recentlyPendingCSRs(indexer cache.Indexer) int {
+func RecentlyPendingCSRs(csrs []certificatesv1beta1.CertificateSigningRequest) int {
```

Comment:
Does this need to be public now?

Reply:
No, two variables should cover it: https://github.com/openshift/cluster-machine-approver/pull/73#discussion_r438039077